packages/runtime-dom/__tests__/getPriorProps.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { getPriorProps } from '../src/patchProp'
+
+describe('runtime-dom: prior props', () => {
+  test('img', () => {
+    const priorProps = getPriorProps('img')
+    expect(priorProps).toHaveLength(1)
+    expect(priorProps).toContain('loading')
+  })
+
+  test('p', () => {
+    const priorProps = getPriorProps('p')
+    expect(priorProps).toHaveLength(0)
+  })
+})
```

Comment:
These tests are not sufficient for this PR.
They do not test the actual behavior of the patch process. Rather, They only ensure your utility function is correct, which doesn't help much.

I suggest adding an E2E test to ensure the loading event never happens for a loading=lazy image.

Reply:
ok,i will do it